.github/workflows/schedule_grype.yaml

```diff
@@ -0,0 +1,12 @@
+name: Grype security scan
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 10 * * * '
```

Comment:
comment: Slightly annoying here is that since I create the `cron` line, I'll get the notifications.

It might be better for someone else to modify this line or, disable and re-enable the workflow, so that they get the notification:

> Notifications for scheduled workflows are sent to the user who initially created the workflow. If a different user updates the cron syntax in the workflow file, subsequent notifications will be sent to that user instead. If a scheduled workflow is disabled and then re-enabled, notifications will be sent to the user who re-enabled the workflow rather than the user who last modified the cron syntax.

https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/notifications-for-workflow-runs

Reply:
That is annoying. Would be better if it was someone on the pipeline team. Happy for you to assign me to receive the notifications or, for me to disable/enable the workflows after merge, so that I get them

Reply:
Side note, would it make sense to schedule it in the early hours of the morning, rather than 10am so the report is ready when we start work?

Reply:
Done :heavy_check_mark:

@lucyb: if you, or someone else, could deactivate/reactivate, that would be great, thanks!